docker-compose.yml

```diff
@@ -281,6 +289,8 @@ services:
     depends_on:
```

Comment:
appeals-service-api-data:
    build:
      context: ./data
      
 Given we're in ./data
 I don't think the adding of package-lock works

Reply:
Yes good spot, I had build from root not checked docker-compose. Sorted in next commit.